pkg/volume/vspherevolume/vsphere_volume_util.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/legacy-cloud-providers/vsphere/vclib"
 )
 
+// List of constants to store fixed values
```

Comment:
drop this comment. It doesn't add anything.

Reply:
According to golint, the const StoragePolicyName must have a comment since it is exported so i thought i can just add a general comment to the constants.

Reply:
I would also prefer either a useful command or no comment at all.